src/components/organisms/Footer.tsx

```diff
@@ -1,5 +1,73 @@
 import React from "react";
+import { CompanyName } from "../../constants";
 
 export const Footer = () => {
-  return <div className="h-80 w-screen p-4 border-4">footer</div>;
+  return (
+    <div>
+      <div className="h-48 w-screen p-4 bg-royalBlue">
```

Comment:
The two div in lines 7 and 8 can be simplified. I would suggest combining them into one and removing the unnecessary props. i.e. `<div className="flex justify-center items-center space-x-1 h-48 w-screen bg-royalBlue ">`

Reply:
A good thing to know is that when writing classnames the leftmost `prop` takes precedence. For example: `className="x y z"` `x` takes precedence.